src/ros_filter.cpp

```diff
@@ -47,9 +47,9 @@ namespace RobotLocalization
 {
   template<typename T>
   RosFilter<T>::RosFilter(ros::NodeHandle nh,
-                          ros::NodeHandle nh_priv,
-                          std::string node_name,
-                          std::vector<double> args) :
+                           ros::NodeHandle nh_priv,
```

Comment:
Er, not sure what happened with this commit, but

(a) it didn't address [my previous comment](https://github.com/cra-ros-pkg/robot_localization/pull/553/commits/520af0707f3f52ef6440c536dab7cc4637f47f2a#r427837341) 
(b) it added a bunch of unnecessary spaces